src/components/browserSupport/components/UpdateIOS.js

```diff
@@ -16,8 +16,8 @@ export default ({ onDismiss }) => (
     title={t`Oops! Your iOS is outdated`}
     text={
       <>
-        <Trans>{'Please update your iOS to access GoodDollar.\nMinimum version required: iOS '}</Trans>
-        {Config.minimalIOSVersion}
+        {t`Please update your iOS to access GoodDollar.
```

Comment:
John Smith, [28.03.2022 16:56]
we should use \n

John Smith, [28.03.2022 16:56]
could you please fix like you fixed the other \ns ?

John Smith, [28.03.2022 16:56]
e.g. wrap to trans and fix styles ?